cmd/apps/rancher_app.go

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/k8s"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallRancher() *cobra.Command {
+	var rancher = &cobra.Command{
+		Use:          "rancher",
+		Short:        "Install rancher",
+		Long:         "Install rancher to manage kubernetes clusters",
+		Example:      "arkade install rancher --hostname mycluster.example.com ",
+		SilenceUsage: true,
+	}
+
+	rancher.Flags().StringP("namespace", "n", "cattle-system", "The namespace to install rancher")
+	rancher.Flags().Bool("update-repo", true, "Update the helm repo")
+	rancher.Flags().String("hostname", "", "public DNS record to access rancher")
+	rancher.Flags().String("ingress-source", "letsEncrypt", "source of TLS cert: letsEncrypt, rancher, secret")
+	rancher.Flags().String("letsencrypt-email", "", "email address used for communication about your certificate")
+	rancher.Flags().Int("replicas", 3, "Number of replicas of rancher pods")
```

Comment:
Is this a production configuration, or min config? Would `1` also be usable?

Reply:
`3` is the default for the helm chart and their usual recommendation for HA, but yeah you can use `1`

All my tests where with `--replicas 1`